src/Sulu/Bundle/AdminBundle/Resources/js/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from 'react';
+import type {ElementRef} from 'react';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import checkboxStyles from './checkbox.scss';
+
+type Props = {
+    checked: boolean,
```

Comment:
@chirimoya Ah. I thought you wanted to rename the `checked` attribute to `value`. Makes sense. I'll add it.